index.js

```diff
@@ -60,14 +60,18 @@ async function isEnabled(options, { request, reply, context }) {
 
 export function federationInfoGraphiQLPlugin({
   federationSchemaUrl = '/federation-schema',
-  version
+  version,
+  umdUrlOverride
```

Comment:
I don''t think this is required. We can just pass the whole structure in the FE and don't use the `federationInfoGraphiQLPlugin` function.

Reply:
Ooh I see your point yeah